Dockerfile

```diff
@@ -469,8 +469,6 @@ ENV CONTAINERD_NAMESPACE=moby
 WORKDIR /go/src/github.com/docker/docker
 VOLUME /var/lib/docker
 VOLUME /home/unprivilegeduser/.local/share/docker
-COPY --link --from=dockercli             /build/ /usr/local/cli
-COPY --link --from=dockercli-integration /build/ /usr/local/cli-integration
 # Wrap all commands in the "docker-in-docker" script to allow nested containers
 ENTRYPOINT ["hack/dind"]
```

Comment:
This one slightly confused me; we don't use the CLI in this stage, or is it always used as "parent" for another stage? (because we _do_ set the `ENTRYPOINT` 🤔

Reply:
I don't think we use the CLI in this stage at all -- we can use it in the next stage which is closer to where it's needed, as I read it.

Reply:
This stage isn't used on its own - it's used as a parent of `dev-systemd-false` or as a parent for `dev-base`  (if the SYSTEMD is false).
The entrypoint is set at this stage to be inherited by the `dev-base` depending on whether we want the systemd or not.